tikv-client/src/main/java/com/pingcap/tikv/types/StringType.java

```diff
@@ -93,4 +93,9 @@ private String convertToString(Object value) throws ConvertNotSupportException {
   protected Object decodeNotNull(int flag, CodecDataInput cdi) {
     return new String((byte[]) super.decodeNotNull(flag, cdi), StandardCharsets.UTF_8);
   }
+
+  @Override
+  public boolean isPushDownSupported() {
+    return true;
```

Comment:
no need to override all types, we can create a default isPushDownSupported function.

Reply:
Yes. I considered that approach but ended a bunch if statement.  - -